mmv1/templates/terraform/examples/parallelstore_instance_baic_beta.tf.tmpl

```diff
@@ -0,0 +1,40 @@
+resource "google_parallelstore_instance" "{{$.PrimaryResourceId}}" {
+  instance_id = "{{index $.Vars "name"}}"
+  location = "us-central1-a"
+  description = "test instance"
+  capacity_gib = 12000
+  network = google_compute_network.network.name
+  file_stripe_level = "FILE_STRIPE_LEVEL_MIN"
+  directory_stripe_level = "DIRECTORY_STRIPE_LEVEL_MIN"
+  deployment_type = "SCRATCH"
+  labels = {
+    test = "value"
+  }
+  provider = google-beta
+  depends_on = [google_service_networking_connection.default]
+}
+
+resource "google_compute_network" "network" {
+  name                    = "{{index $.Vars "network_name"}}"
+  auto_create_subnetworks = true
+  mtu = 8896
+  provider = google-beta
```

Comment:
Standard practice is to have the `provider = google-beta` line at the top of the resource (above the `name`) line. Do you mind making this change for all resources added in this PR?